lib/inference.ts

```diff
@@ -160,6 +160,7 @@ export async function extract({
   chunksSeen,
   chunksTotal,
   requestId,
+  isUsingTextExtract,
```

Comment:
where is this being passed in?

Reply:
being passed in on line 459 in `extractHandler.ts` inside `domExtract`